src/handlers/c_repo_handler.py

```diff
@@ -26,11 +26,14 @@ def __init__(self, config: Config) -> None:
         # It helps in locating the correct files by removing this prefix to the paths found in the error traces.
         self._report_file_prefix = f"{config.PROJECT_NAME}-{config.PROJECT_VERSION.split('-')[0]}/"
  
-        self.repo_local_path = f"{config.GIT_REPO_PATH}/{self._report_file_prefix}"
+        self.repo_local_path = config.GIT_REPO_PATH
         if config.DOWNLOAD_GIT_REPO:
             # downloading git repository for given project
             self.repo_local_path = download_repo(config.GIT_REPO_PATH)
         else:
+            # Override self._report_file_prefix when a local path is provided in GIT_REPO_PATH. 
+            # This is a safer approach because not all packages use the 'package-version' format for the dest folder name.
+            _, self._report_file_prefix = os.path.split(config.GIT_REPO_PATH)
```

Comment:
trying to make sure i understand correctly, so i can set the change on the Pipeline side to support this.

now we just need to set GIT_REPO_PATH=/som/path/downloaded-package-name (could be whatever, just to make sure we have the full path right)?

cuz in this case i will need to introduce a new parameter for the `downloaded-package-name` in the pipeline

the parameter name i mentioned here (`downloaded-package-name`) is just to differ it from the `package-name` itself

example:
`package-name=systemd`
`package-version=257-9`
`downlaoded-package-name=systemd-257`